macro/convertEvtCalc.py

```diff
@@ -130,39 +108,47 @@ def convert_file(infile, outdir):
     else:
         os.system(command)
 
-    infile = f"{outdir}/{fname}"
+    infile  = f"{outdir}/{fname}"
     parsed_data = parse_file(infile)
     outfile = infile.split(".dat")[0] + ".root"
+    ncols   = len(parsed_data[0][2])
+    nvardau = 6 # qualifiers for each daughter
+    remaining_vars = ncols - len(vars_names)
+    
+    if (remaining_vars % nvardau)!=0:
+        raise ValueError(f"- convertEvtCalc - Error: number of daughters is not exact.")
+        sys.exit(1)
```

Comment:
no need to exit, if an exception is raised

Reply:
remnants of some debugging checks. Removed.